decorators/trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.strip
+  end
+end
```

Comment:
- [ ] Your Trimmer Decorator class is not working as intended. Trimming the string ` maximilianus` should return a string of length _10_. Kindly improve your `TrimmerDecorator` class by adding an appropraite method to trim a given string. This should be similar to:

```ruby
class TrimmerDecorator < Decorator
    # The name has a maximum of 10 characters. If it's longer it should trim the word.
    def correct_name
      @nameable.correct_name.length > 10 ? @nameable.correct_name[0..9] : @nameable.correct_name
    end
end
```
**Running your app:**
![image](https://user-images.githubusercontent.com/12745474/187793857-54ef495d-4459-4800-9b66-4aef4ee3f7d3.png)